auth_ldaps/models/res_company_ldap.py

```diff
@@ -1,4 +1,5 @@
 # Copyright (C) Creu Blanca
+# Copyright (C) 2018 Brainbean Apps
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 import ldap
```

Comment:
There was an issue regarding this import: https://github.com/OCA/server-auth/issues/37
Can you fix it by protecting the import?
```python
try:
    import ldap
except (ImportError) as err:
    _logger.debug(err)
```
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#importerror

Reply:
Done